gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -0,0 +1,1541 @@
+    )
+
+    parser.add_argument(
+        "-p",
+        "--platform",
+        help="Blockchain platform like eth-ethereum",
+        dest="platform",
+        default="eth-ethereum",
+        type=str,
+        choices=PLATFORMS,
+    )
+
+    parser.add_argument(
+        "-t", "--top", default=20, dest="top", help="Limit of records", type=int
+    )
+
+    parser.add_argument(
+        "-s",
+        "--sort",
+        dest="sortby",
+        type=str,
+        help="Sort by given column. Default: index",
+        default="index",
+        choices=["index", "id", "type", "active", "address"],
+    )
+
+    parser.add_argument(
+        "--descend",
+        action="store_false",
+        help="Flag to sort in descending order (lowest first)",
+        dest="descend",
+        default=True,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = paprika.get_contract_platform(ns_parser.platform)
+
+        if df.empty:
+            print(f"Nothing found for platform: {ns_parser.platform}")
+            return
+
+        df = df.sort_values(ns_parser.sortby, ascending=ns_parser.descend)
+        print(
+            tabulate(
+                df.head(ns_parser.top),
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+# Coin Related
+
+
+def find(other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        prog="find",
+        add_help=False,
+        description="""
+        Find similar coin by coin name,symbol or id. If you don't remember exact name or id of the Coin at CoinPaprika,
+        you can use this command to display coins with similar name, symbol or id to your search query.
+        Example of usage: coin name is something like "kusama". So : find -c kusama -k name -t 25
+        It will search for coin that has similar name to kusama and display top 25 matches.
+        -c, --coin stands for coin - you provide here your search query
+        -k, --key it's a searching key. You can search by symbol, id or name of coin
+        -t, --top it displays top N number of records.
+        """,
+    )
+    parser.add_argument(
+        "-c",
+        "--coin",
+        help="Coin name or id, or symbol",
+        dest="coin",
+        required=False,
+        type=str,
+    )
+
+    parser.add_argument(
+        "-k",
+        "--key",
+        dest="key",
+        help="Specify by which column you would like to search: symbol, name, id",
+        type=str,
+        choices=["id", "symbol", "name"],
+        default="name",
+    )
+
+    parser.add_argument(
+        "-t", "--top", default=10, dest="top", help="Limit of records", type=int
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
```

Comment:
@DidierRLopes thanks for the review. Don't apologize :D I am happy that you are picky! 

Regarding this one with 
`if other_args:
       if "-" not in other_args[0]:
           other_args.insert(0, "-c")
`
I can only add this one in `load` view because in `search` or `find` view
there will be issue with it, because in coinpaprika ids of coins, exchanges, or platforms have format `ksm-kusama`, `btc-bitcoin`, `uni-uniswap` and so on. 
So if I add this in search and find view there will be an issue if the user will try to do a search using id of coin or exchange like :
```
(✨) (crypto)>(cp)> search eth-ethereum
usage: search -q QUERY [-c {currencies,exchanges,icos,people,tags,all}]
              [-t TOP] [-s {category,id,name}] [--descend] [-h]
search: error: the following arguments are required: -q/--query
The command selected doesn't exist
```

Reply:
What about doing something like:
```
if other_args:
       if other_args[0][0] == '-':
           other_args.insert(0, "-c")
```

This way you are looking for the first character of the string rather than just if a string contains a "-".
PS: Didn't test this, but in theory seems like it should work.

Reply:
Thanks :) I test it and worked, but should be with `not` 🤯  

`if other_args:
       if not other_args[0][0] == '-':
           other_args.insert(0, "-c")`

Reply:
Aww yes, forgot the not! Ups! Glad it worked eheh